src/database/operations/dbOperationAuditLogs/createDbOperationAuditLog.ts

```diff
@@ -0,0 +1,18 @@
+import TinyPg from 'tinypg';
+import { AuthContext } from '../../../types';
+
+export const createDbOperationAuditLog = async (
```

Comment:
We should avoid acronyms in function names (`database` is more clear and explicit than `db`)

Reply:
The function name follows the database table name. [See above](https://github.com/PhilanthropyDataCommons/service/pull/1615#discussion_r2047663030), we can continue to iterate on names there.